guides/common/assembly_configuring-satellite-custom-server-certificate.adoc

```diff
@@ -25,8 +25,14 @@ To configure your {ProjectServer} with a custom certificate, complete the follow
 . If you have external {SmartProxyServers} registered to {ProjectServer}, configure them with custom SSL certificates.
 For more information, see {InstallingSmartProxyDocURL}configuring-capsule-custom-server-certificate_{smart-proxy-context}[Configuring {SmartProxyServer} with a Custom SSL Certificate] in _{InstallingSmartProxyDocTitle}_.
 
+:ProductName: {ProjectServer}
```

Comment:
This is defined at the parent level (in `master.adoc`). So let's remove `:ProductName:` and `:!ProductName:` in this assembly. It should fix the build of the _Installing Server_ guide.